packages/arb-token-bridge-ui/src/components/TransferPanel/WithdrawalConfirmationDialog.tsx

```diff
@@ -72,15 +68,11 @@ export function WithdrawalConfirmationDialog(
   const [checkbox2Checked, setCheckbox2Checked] = useState(false)
 
   const { isArbitrumOne } = isNetwork(l2.network.id)
-  const baseChainId = getBaseChainIdByChainId({
```

Comment:
I simplified this because it can be called inside `getTxConfirmationDate`. We don't have to pass it as a prop.